libraries/AP_HAL_Linux/VideoIn.cpp

```diff
@@ -41,6 +42,29 @@ extern const AP_HAL::HAL& hal;
 
 using namespace Linux;
 
+int VideoIn::get_fd(void)
+{
+    return _fd;
+}
+
```

Comment:
I assume you are planning to use that in the next PRs but could you describe what you want to do with the fd ?